### PR TITLE
C12567 convert hybrid

### PR DESCRIPTION
**Pull request by jkrank**

### CODE REVIEW

File: Entity-SQL.js [hunk 1]
```diff
@@ -455,6 +455,7 @@ module.exports.define("keyChange", function (field, old_val) {
             this.duplicate_key_message = this.getMessageManager().add({
                 type: "E",
                 fixed: true,
+                text: "key value conflicts with another record: " + temp_key,
```
Comment (stephenfrancis) [inline Entity-SQL.js]: why is this required? duplication?

File: fields_supl/Flex.js [hunk 2]
```diff
@@ -234,6 +234,9 @@ module.exports.override("setFromResultSet", function (resultset) {
         }
     } catch (e) {
         this.report(e);
+        print(this.owner);
```
Comment (stephenfrancis) [inline fields_supl/Flex.js]: can you use this.output() rather than print()?

**Comment by jkrank** [comment]
I've removed the text content, but the attribute itself was added because otherwise I got error complaining that text is missing
